vmc/resource_vmc_cluster.go

```diff
@@ -53,12 +53,7 @@ func resourceCluster() *schema.Resource {
 				Optional:    true,
 				Description: "The instance type for the esx hosts added to this cluster.",
 				ValidateFunc: validation.StringInSlice(
-					[]string{HostInstancetypeI3, HostInstancetypeR5}, false),
-			},
-			"storage_capacity": {
```

Comment:
curious to know why storage_capacity is no longer needed

Reply:
storage capacity is only needed for R5 metal host instance type. R5 is being deprecated moving forward. We will also have to remove support for R5 from cluster and SDDC resource.

Reply:
what about customers who have access to R5 hosts?

Reply:
We have not tested R5 cluster creation ourselves. We can keep it but we have not tested it ourselves.